### DISCUSSION TITLE
Vanilla Extract with Qwik

### DISCUSSION THREAD

**Discussion by brianbento** [vanilla-extract-css/vanilla-extract#880] [Q&A]
Been trying out Qwik (https://qwik.builder.io/) and I've been trying to get vanilla-extract working with it.

It uses Vite under the hood and was wondering if anyone has got it working.

**Comment by n8sabes** [comment]
@brianbento, Check out the plugin written by @wmertens.

[vanilla-qwik-vite-plugin.ts](https://stackblitz.com/edit/qwik-ve-css-export?file=vanilla-qwik-vite-plugin.ts)

**Reply by wmertens** [reply]
To clarify, this is a poc plugin that exports the css as a default export so that it can be included only when you use the corresponding component.

It's not entirely working but I have ideas on how to fix that and it will be part of the next release of https://GitHub.com/wmertens/qwik-styled-ve

**Comment by wmertens** [comment]
The plugin is now exported from `styled-vanilla-extract` (I renamed the package). NPM install it and import from `style-vanilla-extract/vite` for the plugin. 

See https://github.com/wmertens/styled-vanilla-extract/blob/main/README.md
